service/src/test/java/bio/terra/workspace/service/workspace/ApplicationServiceTest.java

```diff
@@ -195,6 +222,58 @@ public void applicationEnableTest() {
     wsmApp = appService.getWorkspaceApplication(USER_REQUEST, workspaceId2, LEO_ID);
     assertEquals(LEO_APP, wsmApp.getApplication());
     assertTrue(wsmApp.isEnabled());
+
+    // validate decommissioned apps can't be re-enabled
+    // Clear error list first
+    appService.enableTestMode();
```

Comment:
Seems like you either put this here or in the `@BeforeEach`, but not both

Reply:
yeah - I had been using it to clear the error list, which is a little hacky. A couple different operations (creates, enables, etc.) write to the error log so I wanted to avoid grabbing magic indices out of `appService.getErrorList()` (side note: should only errors write to this log, or can we rename it to `getLogList`?)

I think a better way is just getting the most recent log, switched to that.